docs/root/intro/version_history.rst

```diff
@@ -32,6 +30,10 @@ Version history
 * tracing: the sampling decision is now delegated to the tracers, allowing the tracer to decide when and if
   to use it. For example, if the :ref:`x-b3-sampled <config_http_conn_man_headers_x-b3-sampled>` header
   is supplied with the client request, its value will override any sampling decision made by the Envoy proxy.
+ * admin-console mutations must be sent as POSTs, rather than GETs. Mutations include:
```

Comment:
Needs to be alpha sorted.

Reply:
Please also prefix with "admin: " like the other entry.